library/src/main/java/pl/rspective/voucherify/android/client/model/PublishEntry.java

```diff
@@ -0,0 +1,28 @@
+package pl.rspective.voucherify.android.client.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class PublishEntry {
+
+    private String customer;
+    private String channel;
+    
+    @SerializedName("published_at")
+    private Date publishedAt;
+
```

Comment:
Oops, another thing behind current state... You won't see it in Node SDK either, since JS doesn't need an object model and examples also don't include publications with metadata. Anyway, we should update it here and in Java SDK.